korman/korlib/__init__.py

```diff
@@ -29,6 +29,30 @@
         msg = "Korlib C Module did not load correctly."
     print(msg, "Using PyKorlib :(", sep=' ')
 
+    def create_funky_ramp(mipmap, additive=False):
+        kLUTHeight = 16
+        kLUTWidth = 16
+
+        buf = bytearray(kLUTHeight * kLUTWidth * 4)
+
+        for i in range(kLUTHeight):
+            for j in range(kLUTWidth):
+                x = j / (kLUTWidth - 1);
+                y = i / (kLUTHeight - 1);
+
+                start = i*j*4
+                end = start + 4
+
+                if additive:
+                    if x < y:
+                        x = y
+
+                    buf[start:end] = [b for b in (255, 255, 255, int(x * 255.9))]
+                else:
+                    buf[start:end] = [b for b in (255, 255, 255, int((x * y) * 255.9))]
```

Comment:
@Hoikas This buffer stuff is wrong, but my Python isn't great and I'm not seeing anything obvious. The C++ implementation works.